stonesoup/models/measurement/tests/test_models.py

```diff
@@ -562,3 +563,67 @@ def test_inverse_function():
     assert approx(inv_measure_state[3], 0.02) == 17.10
     assert approx(inv_measure_state[4], 0.02) == 1736.48
     assert approx(inv_measure_state[5], 0.02) == 17.36
+
+
+def test_binning():
+    real_state = State(state_vector=StateVector([10e3, 100., 10e3, 100., 10e3, 100.]))
+
+    measurement_model = RangeRangeRateBinning(range_res=10,
+                                              range_rate_res=5,
+                                              ndim_state=6,
+                                              mapping=[0, 2, 4],
+                                              velocity_mapping=[1, 3, 5],
+                                              noise_covar=np.array([100,
+                                                                   10,
+                                                                   np.pi/18,
+                                                                   np.pi/18]))
+
+    measured = measurement_model.function(real_state, noise=True)
+    assert (measured[0, 0]/measurement_model.range_res).is_integer()
+    assert (measured[1, 0]/measurement_model.range_rate_res).is_integer()
+
+
+def test_binning_pdf():
+    real_state = State(state_vector=StateVector([10e3, 100., 10e3, 100., 10e3, 100.]))
+
+    measurement_model = RangeRangeRateBinning(range_res=10,
+                                              range_rate_res=5,
+                                              ndim_state=6,
+                                              mapping=[0, 2, 4],
+                                              velocity_mapping=[1, 3, 5],
+                                              noise_covar=np.array([100,
+                                                                   10,
+                                                                   np.pi/18,
+                                                                   np.pi/18]))
+
+    measured = measurement_model.function(real_state, noise=True)
+    pdf = measurement_model.pdf(State(measured), real_state)
+    assert pdf != 0
```

Comment:
Should there be a test (not necessarily here) that check the pdf value, maybe against a manual calculation?